src/pages/SwapPage/SwapPage.tsx

```diff
@@ -70,10 +70,11 @@ const SwapPage: React.FC = () => {
         if (txns) {
           const filteredTxns = txns.filter(
             (txn) =>
-              !transactions.filter(
+              !transactions.find(
                 (tx) => tx.transaction.id === txn.transaction.id,
-              ).length,
+              ),
           );
+          console.log('ccc', filteredTxns);
```

Comment:
Remove this log message and the other one that says bbb which got merged to master

Reply:
Removed the console logs and pushed commit